docs/migration_guide.md

```diff
@@ -14,61 +14,83 @@ Since the migration might alter the database schema and the settings of `harbor.
 - With the introduction of storage and artifact quotas in version 1.9.0, migration from 1.7.x and 1.8.x might take a few minutes. This is because the `core` walks through all blobs in the registry and populates the database with information about the layers and artifacts in projects.
 - With the introduction of storage and artifact quotas in version 1.9.0, replication between version 1.9.0 and a previous version of Harbor does not work. You must upgrade all Harbor nodes to 1.9.0 if you have configured replication between them.
 
+
+
+
+
 ## Upgrading Harbor and Migrating Data
 
-1. Log in to the host that Harbor runs on, stop and remove existing Harbor instance if it is still running:
+#### Shutdown Harbor Service
 
-    ```sh
-    cd harbor
-    docker-compose down
-    ```
+Log in to the host that Harbor runs on, stop and remove existing Harbor instance if it is still running:
 
-2. Back up Harbor's current files so that you can roll back to the current version if necessary.
+```sh
+cd harbor
+docker-compose down
+```
 
-    ```sh
-    mv harbor /my_backup_dir/harbor
-    ```
+#### Backup Harbor Data
 
-    Back up database (by default in directory `/data/database`)
+Back up Harbor's current files so that you can roll back to the current version if necessary.
 
-    ```sh
-    cp -r /data/database /my_backup_dir/
-    ```
+```sh
+mv harbor /my_backup_dir/harbor
+```
+
+Back up database (by default in directory `/data/database`)
 
-3. Get the latest Harbor release package from Github:
-   [https://github.com/goharbor/harbor/releases](https://github.com/goharbor/harbor/releases)
+```sh
+cp -r /data/database /my_backup_dir/
+```
 
-4. Before upgrading Harbor, perform migration first.  The migration tool is delivered as a docker image.
+#### Get Package of Latest Harbor Installer
+
+You can download it in this url [https://github.com/goharbor/harbor/releases](https://github.com/goharbor/harbor/releases).
+
+#### Migrating Config file
+
+Before upgrading Harbor, perform migration first. The migration tool is delivered as a docker image.
+
+1. **Online installer** need pull the migration image from docker hub. Replace [tag] with the release version of Harbor (e.g. v1.5.0) in the below command:
```

Comment:
Pull the migration image from Docker Hub. Replace `[tag]` with the Harbor release version, for example, `v1.5.0`, in the following command: